CHANGELOG-1.0.md

```diff
@@ -0,0 +1,121 @@
+# Release notes for master
+
+[Documentation](https://docs.k8s.io/docs/home)
+
+# Changelog since 0.5.0
+
+## Changes by Kind
+
+### Feature
+
+- Adding configurable QPS and Burst to k8s client and use a separate client for leader election. ([#91](https://github.com/kubernetes-csi/external-resizer/pull/91), [@RaunakShah](https://github.com/RaunakShah))
+- Pods watch is an expensive operation. Hence make the feature of handling volume in use errors optional, by registering pod informer based on a feature gate ([#89](https://github.com/kubernetes-csi/external-resizer/pull/89), [@saikat-royc](https://github.com/saikat-royc))
```

Comment:
We used flag instead of feature gate right? Can we document the flag here? 

Also should we combine it with the note about #86?

Reply:
It is documented in the readme and yeah we can combine these two under one chanelog.

Reply:
done.